src/managers/testing_manager.ts

```diff
@@ -129,7 +127,8 @@ export class TestingManager {
 				// The column features are names of attributes we expect to find having values true or false
 				tTokens.forEach((iToken, iIndex) => {
 					if (iToken.formula !== '') {
-						if (iCase.values[iToken.name]) {
+						// Codap v3 currently returns strings, even for booleans, so we have to compare the string for now
+						if (iCase.values[iToken.name] === "true") {
```

Comment:
If you wanted to be forwards compatible, you could change this to:
```suggestion
						if (iCase.values[iToken.name] === "true" || iCase.values[iToken.name] === true) {
```